aten/src/ATen/native/quantized/cpu/q_avgpool.cpp

```diff
@@ -375,7 +375,7 @@ Tensor qnnpack_avg_pool2d(
   CAFFE_ENFORCE(
       setupStatus == pytorch_qnnp_status_success,
       "failed to setup QNNPACK Average Pooling operator");
-  pthreadpool_t threadpool = caffe2::mobile_pthreadpool();
+  pthreadpool_t threadpool = caffe2::pthreadpool_();
```

Comment:
The function name ending in underscore is for internal usage - it returns the pthreadpool_t handle of the underlying implementation.  Open to new names if you have suggestions.